### PR TITLE
Update Ansible script to install VS2017 for JDK11

### PR DESCRIPTION
**Pull request by ChengJin01**
The changes are to install VS2017 so as to compile
JDK11 on Windows.

Signed-off-by: CHENGJin <jincheng@ca.ibm.com>

### CODE REVIEW

**Comment by ChengJin01** [comment]
FYI:  @jdekonin

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # .\ConfigureRemotingForAnsible.ps1 -ForceNewSSLCert
 # .\ConfigureRemotingForAnsible.ps1 -SkipNetworkProfileCheck
 
-- hosts: "{{ groups['Vendor_groups'] | default('build*win*:test*win*') }}"
+- hosts: all
```
Comment (jdekonin) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml]: Please undo this line.  This topic needs to be addressed in another issue which I'll open now.

Comment (ChengJin01) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml]: Already reverted.

**Comment by ChengJin01** [comment]
> The download link could be changed to point directly at the download file, https://aka.ms/vs/15/release/vs_community.exe...

Already updated to the download link.

**Comment by ChengJin01** [comment]
> I'm assuming that this visual studio will sit cleanly along the others? 

I am not sure what you meant. But VS2017 is installed in a separate directory different from 2010/2013.

> ...but are the spaces in windows paths going to cause a problem?

According to the command above   **win_command: 'regsvr32 /s "{{ item }}"'**,
it should work as the item is already quoted in there.

**Comment by karianna** [comment]
Please note that Ansible tower will still need to run this to update our windows hosts.  

@gdams - this is on a cron now correct?

**Comment by gdams** [comment]
not for windows!

**Pull request merged**